src/NonceManager.sol

```diff
@@ -1,12 +1,13 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.23;
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.29;
 
 import {INonceManager} from "./interfaces/INonceManager.sol";
 import {BaseAuthorization} from "./BaseAuthorization.sol";
 
 /// @title NonceManager
 /// @notice A contract that manages nonces to prevent replay attacks
 abstract contract NonceManager is INonceManager, BaseAuthorization {
+    /// @notice A mapping of keys to their corresponding sequence numbers
```

Comment:
move natspec for public/external functions/storage to interface and inheritdoc

Reply:
i just removed, i wasn't sure how to do it for storage var, and this one was not rly useful